mp/src/vgui2/vgui_controls/CvarComboBox.cpp

```diff
@@ -9,29 +9,28 @@ using namespace vgui;
 
 Panel *Create_CvarComboBox()
 {
-    return new CvarComboBox( nullptr, nullptr );
+    return new CvarComboBox( nullptr, nullptr, nullptr );
 }
 
 //not sure which build factory to use here (and if the constructor above is necessary)
 DECLARE_BUILD_FACTORY_CUSTOM( CvarComboBox, Create_CvarComboBox );
 
-CvarComboBox::CvarComboBox(Panel *parent, const char *panelName, int numLines, bool allowEdit, char const *cvarname, bool ignoreMissingCvar)
-    : ComboBox(parent, panelName, numLines, allowEdit), m_cvar((cvarname) ? cvarname : "", (cvarname) ? ignoreMissingCvar : true)
+CvarComboBox::CvarComboBox(Panel *parent, const char *panelName, char const *cvarname)
+    : ComboBox(parent, panelName, 5, false), m_cvar(cvarname)
```

Comment:
> Does this cause a lot of warnings about missing cvars in console?

Nope

> we manually declare them before the res file gets loaded

That's true yeah. Figured it doesn't make much sense to declare this in a res file as items need to be added to it anyhow, which I think can only be done through code. 

> anything initialized via res file (using that `Create_CvarComboBox` guy) will now yell.

Yeah I'm not sure how to get around this given what I assumed above.

Reply:
> Yeah I'm not sure how to get around this given what I assumed above.

We don't need to, I was just concerned about warnings in this branch, like booting up the game and scoping the console for any, because that points to panels being constructed via res file, which we'd want to fix.

Reply:
I compared with develop and there are no extra warnings. Searched through the code for references and caught everything by the looks of it.